include/mbedtls/build_info.h

```diff
@@ -74,7 +74,11 @@
 #include MBEDTLS_USER_CONFIG_FILE
 #endif
 
-#if defined(MBEDTLS_PK_C) && defined(MBEDTLS_USE_PSA_CRYPTO)
+/* The PK wrappers need pk_write functions to format RSA key objects
+ * when they are dispatching to the PSA API. This happens under USE_PSA_CRYPTO,
+ * and also even without USE_PSA_CRYPTO for mbedtls_pk_sign_ext(). */
```

Comment:
I've now realized that the bit about `mbedtls_pk_sign_ext` is a [regression in 3.2.0](https://github.com/Mbed-TLS/mbedtls/issues/6408). I'm going to make a separate pull request for that, then rebase this one on top of it.

Reply:
The regression was fixed and this PR was rebased, so I'm marking this conversation as resolved.